lib/exe/css.php

```diff
@@ -34,98 +35,125 @@ function css_out(){
 
     if ($INPUT->str('s') == 'feed') {
         $mediatypes = array('feed');
-        $type = 'feed';
     } else {
-        $mediatypes = array('screen', 'all', 'print');
-        $type = '';
+        $mediatypes = array('screen', 'all', 'print', 'handheld');
```

Comment:
'feed' is gone, it was for making a specific cachename. I did not look how it works now. Depending on the included different css file name for the rss feed?